src/stdune/list.ml

```diff
@@ -1,7 +1,7 @@
-type 'a t = 'a list
-
+open ListType
 include ListLabels
 
+type nonrec 'a t = 'a t
```

Comment:
I was surprised that this works, given that the type is still defined twice in this module (once through the include and once here). Then I remembered that this is a new feature in 4.08, so simply switching the order of the `include` and `type 'a t = 'a list` should be enough.